eventra-saas/ops/tools/rollback-ci.sh

```
@@ -0,0 +1,172 @@
+    echo "Aborting."
+    exit 0
+  fi
+fi
+
+# Helper to run or echo commands
+run() {
+  if [ "$DRY_RUN" = true ]; then
+    echo "[DRY-RUN] $*"
+  else
+    echo "[RUN] $*"
+    eval "$@"
```

Shellcheck (0.11.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)